app/controllers/application_controller.rb

```diff
@@ -3,7 +3,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_current_user
-  before_action :access_control
+  before_action :restrict_to_employee
```

Comment:
Note that this action can't do anything yet because `#set_current_user` redirects to `SessionsController#unauthenticated` when `@current_user` isn't `present?`

Reply:
It's changed in a later branch. But I may as well tweak set_current_user now.